keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -25,32 +25,25 @@
 class RandomResizedCrop(BaseImageAugmentationLayer):
     """Randomly crops a part of an image and resizes it to provided size.
 
-    This implementation takes an intuitive approach, where we crop the images to a
-    random height and width, and then resize them. To do this, we first sample a
-    random value for area using `crop_area_factor` and a value for aspect ratio using
-    `aspect_ratio_factor`. Further we get the new height and width by
-    dividing and multiplying the old height and width by the random area
-    respectively. We then sample offsets for height and width and clip them such
-    that the cropped area does not exceed image boundaries. Finally we do the
-    actual cropping operation and resize the image to `target_size`.
+    This implementation takes a distortion-oriented approach, which means the
+    amount of distortion in the image is proportional to the `zoom_factor`
```

Comment:
Probable error in the docstring:
The amount of distorsion is proportional to aspect_ratio_factor, not the zoom_factor. Distorsion means changing the aspect ratio, not zooming.